x-pack/plugins/cases/public/components/user_action_tree/index.tsx

```diff
@@ -568,19 +568,24 @@ export const UserActionTree = React.memo(
       ]
     );
 
-    const bottomActions = [
-      {
-        username: (
-          <UserActionUsername username={currentUser?.username} fullName={currentUser?.fullName} />
-        ),
-        'data-test-subj': 'add-comment',
-        timelineIcon: (
-          <UserActionAvatar username={currentUser?.username} fullName={currentUser?.fullName} />
-        ),
-        className: 'isEdit',
-        children: MarkdownNewComment,
-      },
-    ];
+    const bottomActions = userCanCrud
```

Comment:
A readonly user will not be able to add comments so remove the comment section from the page.